ffi/java/android/project/mcl-android/build.gradle

```diff
@@ -0,0 +1,101 @@
+plugins {
+	id 'com.android.library'
+	id 'maven-publish'
+	id 'com.jfrog.bintray'
+}
+
+android {
+	compileSdkVersion 30
+	buildToolsVersion "29.0.3"
+
+	defaultConfig {
+		minSdkVersion 16
+		targetSdkVersion 30
+		versionCode 1
```

Comment:
I think we should align the versionCode and versionName with the library version (currently v1.29)